### PR TITLE
Decorate controllers and removed unused ones

### PR DESCRIPTION
**Pull request by ouranos**

### CODE REVIEW

**Comment by ouranos** [comment]
@hedudelgado FYI

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/app_instances_controller.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  #==================================================================
+  # Instance methods
+  #==================================================================
+  # GET /mnoe/jpi/v1/organization/1/apps.json?timestamp=151452452345
+  def index
+    @app_instances = parent_organization.app_instances.select do |i|
+      i.active? && i.updated_at > Time.at(timestamp) && can?(:access,i)
+    end
+  end
```
Comment (BrunoChauvet) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/app_instances_controller.rb]: Could we improve this?

```ruby
@app_instances = parent_organization.app_instances.where(active: true).where("created_at >= ?", Time.at(timestamp)).select do |i|
  can?(:access,i)
end
```

Comment (ouranos) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/app_instances_controller.rb]: Good point. I'll see if I can merge #108 first

Comment (ouranos) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/app_instances_controller.rb]:
```ruby
@app_instances = parent_organization.app_instances.active.where("updated_at.gt" => Time.at(timestamp)).select do |i|
  can?(:access,i)
end
```

**Comment by ouranos** [comment]
@BrunoChauvet rebased after perf improvements from #108

**Pull request merged**